src/docs/previews/popover/main/tailwind/index.svelte

```diff
@@ -6,7 +6,9 @@
 	const {
 		elements: { trigger, content, arrow, close },
 		states: { open },
-	} = createPopover();
+	} = createPopover({
+		forceVisible: true,
+	});
```

Comment:
Same here

Reply:
EDIT: I think the git history is a bit weird for whatever reason 🤔

Reply:
Indeed, I should have followed your contributing guide like I did for my first pull request 😅(great guide btw)

I tried resetting my fork but it auto-closed this PR and I couldn't get it to work anyway so I created a new PR with just one commit this time.

Sorry for the mess !

Reply:
No worries, thank you for the hard work!!